walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,205 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.ArrayList;
 
 public final class FindMeetingQuery {
+
+  // Constants for 0 and the number of minutes in a day. 
+  // 0 means 0 minute since start of a day. 
+  // 1440 is the total number of minutes in a day, which marks the end of a day. 
+  private static final int START_OF_DAY = 0; 
+  private static final int END_OF_DAY = 1440; 
```

Comment:
Same here.

END_OF_DAY_MINUTES

Also, in this case we can perform the computation so it's clear how this value came about.

ex:

END_OF_DAY_MINUTES = 24 * 60;

Reply:
Updated.